README.md

```diff
@@ -189,10 +189,51 @@ katib-ui-5767cfccdc-pwg2x           1/1     Running     0          36s
 
 For the Katib Experiments check the [complete examples list](./examples/v1beta1).
 
+# Quickstart
+
+You can run your first HyperParameter Tuning Experiment using [Katib Python SDK](./sdk/python/v1beta1).
+
+In the following example we are going to maximize a simple objective function:
+$F(a,b) = 4a - b^2$. The bigger $a$ and the lesser $b$ value, the bigger the function value $F$.
+
+```python
+import kubeflow.katib as katib
+
+# Step 1. Create an objective function.
+def objective(parameters):
+    # Import required packages.
+    import time
+    time.sleep(5)
+    # Calculate objective function.
+    result = 4 * int(parameters["a"]) - float(parameters["b"]) ** 2
+    # Katib parses metrics in this format: <metric-name>=<metric-value>.
+    print(f"result={result}")
+
+# Step 2. Create HyperParameter search space.
+parameters = {
+    "a": katib.int(min=10, max=20),
+    "b": katib.double(min=0.1, max=0.2)
```

Comment:
This doesn't seem Pythonic to me. Is it easy to support user specified list ranges, e.g. `range(10, 20)`?

Reply:
@terrytangyuan Let's continue discussion around it here: https://github.com/kubeflow/katib/pull/1951#discussion_r970890433.